eks-anywhere-common/Addons/Partner/Kasten/K10/helm.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
```

Comment:
Please remove Prometheus and all Observability pods which is added to this helm release.